internal/controller/pipelinerun_helper_test.go

```diff
@@ -0,0 +1,294 @@
+package controller
+
+import (
+	"encoding/json"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	v1 "knative.dev/pkg/apis/duck/v1"
+)
+
+var _ = Describe("Unit testing for pipelinerun_helper", func() {
+	var (
+		testTruePipelineRun, testFalsePipelineRun, testErrMmarshelPipelineRun *tektonv1.PipelineRun
+		notPipelineRun                                                        *tektonv1.TaskRun
+	)
+
+	Describe("Push Pipelinerun Ended with finalizer, annotation and results", func() {
```

Comment:
Let's add some commas/semi-colons or something to make sure the description is read in the way we intended to (same for other test cases as well).
```suggestion
	Describe("Push Pipelinerun; ended; with finalizer, annotation and results", func() {
```

Reply:
Done